Web/HTML1/Assignment1/Index.html

```diff
@@ -0,0 +1,234 @@
+
+<!DocTyoe html>
+
+<html>
+
+<head>
+
+<title>Account Creation</title>
+</head>
+
+<body>
+	<table>
+		<tr>
+			<td colspan="2"><b>Create Your Account:</b></td>
+		</tr>
+
+		<tr>
+			<td align="right">Email ID:</td>
```

Comment:
thats for validation and in this assignment we just had to make frames..so no asterisk sign... if its needed and mandatory..we will use 'required'  in html5..